charts/sonarqube-dce/templates/networkpolicy.yaml

```diff
@@ -181,7 +181,13 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 spec:
+{{- if.Values.networkPolicy.additionalNetworkPolicys -}}
```

Comment:
We should revert the order here. If people set the new value `.Values.networkPolicy.additionalNetworkPolicies` this should take precedence over the deprecated one, right?

Reply:
Hm i usually prefer the other way around for compatibility

Reply:
Ok this seems to be the standard in our charts. Thanks for clarifying :)